source/Octopus.Data/Model/ReferenceCollection.cs

```diff
@@ -61,7 +61,7 @@ public void ReplaceAll(IEnumerable<T> newItems)
 
         public override string ToString()
         {
-            return string.Join(", ", this.Select(x => x.ToString()));
+            return string.Join(", ", this.Where(x => x != null).Select(x => x?.ToString()));
```

Comment:
Could we use `.Select(x => x!.ToString())` here instead?

Reply:
Done 👍